packages/thirdweb/src/react/web/ui/ConnectWallet/Details.tsx

```diff
@@ -983,3 +987,31 @@ function SwitchNetworkButton(props: {
     </Button>
   );
 }
+
+function SwitchMetamaskAccount(props: {
+  closeModal: () => void;
+}) {
+  const wallet = useActiveWallet();
+  const connectLocale = useConnectUI().connectLocale;
+  const injectedMetamaskProvider = injectedProvider("io.metamask");
```

Comment:
probably better to only run this once we know the wallet id, and then do another level of check for this -> that way this doesn't run if the activeWallet isn't metamask first

Reply:
Done